kinto/core/storage/postgresql/__init__.py

```diff
@@ -737,14 +739,28 @@ def _format_conditions(self, filters, id_field, modified_field,
 
             if is_like_query:
                 # Operand should be a string.
-                # Add implicit start/end wildchars if none is specified.
+                # Add implicit start/end wildcards if none is specified.
                 if '*' not in value:
                     value = '*{}*'.format(value)
                 value = value.replace('*', '%')
 
             if filtr.operator == COMPARISON.HAS:
                 operator = 'IS NOT NULL' if filtr.value else 'IS NULL'
                 cond = '{} {}'.format(sql_field, operator)
+
+            elif filtr.operator == COMPARISON.CONTAINS_ANY:
+                value_holder = '{}_value_{}'.format(prefix, i)
+                holders[value_holder] = value
+                # Postgres's && operator doesn't support jsonbs.
+                # However, it does support Postgres arrays of any
+                # type. Assume that the referenced field is a JSON
+                # array and convert it to a Postgres array.
+                # N.B. This won't work on JSON objects.
```

Comment:
What does it mean "won't work" ? "will always be false" ?

Reply:
If you've got an array of json objects it won't work. The array should be only strings or int.

Reply:
Oh I just understood your remark, I guess we should add a test to make sure of it.